index.html

```diff
@@ -68,108 +68,58 @@ <h2>More</h2>
                 </div>
-                <div class="whats-happening-box dashed-border-green two-rows__expand-one">
-                    <div class="two-columns__expand-two">
-                        <img class="dashed-border-pink" src="" alt="profile-pic">
-                        <form contenteditable="true" class="dashed-border-pink" action="whats-happening"><p><strong><span class="style-three">What's happening?</span></strong></p></form>
-                    </div>
-                    <div class="two-columns__expand-one dashed-border-pink">
-                        <ul class="five-columns">
-                            <li>
-                                <img class="small-icon" src="" alt="picture">
-                            </li>
-                            <li>
-                                <img class="small-icon" src="" alt="gif">
-                            </li>
-                            <li>
-                                <img class="small-icon" src="" alt="graph">
-                            </li>
-                            <li>
-                                    <img class="small-icon" src="" alt="smiley">
-                            </li>
-                            <li>
-                                <img class="small-icon" src="" alt="calendar">
-                            </li>
-                        </ul>
-                        <button class="tweet-button-light-small">Tweet</button>
-                    </div>
+                <div class="whats-happening-box grey-border dashed-border-green">
+                    <img class="dashed-border-pink profile-pic" src="" alt="profile-pic">
+                    <form contenteditable="true" class="whats-happening-form dashed-border-pink" action="whats-happening"><p><span class="style-three">What's happening?</span></p></form>
+                    <ul class="five-columns icons-list-home">
+                        <li>
+                            <img class="small-icon" src="" alt="picture">
+                        </li>
+                        <li>
+                            <img class="small-icon" src="" alt="gif">
+                        </li>
+                        <li>
+                            <img class="small-icon" src="" alt="graph">
+                        </li>
+                        <li>
+                            <img class="small-icon" src="" alt="smiley">
+                        </li>
+                        <li>
+                            <img class="small-icon" src="" alt="calendar">
+                        </li>
+                    </ul>
+                    <button class="tweet-button-light-small">Tweet</button>
                 </div>
-                <div class="grey-space dashed-border-green"></div>
-                <div class="main-content dashed-border-green">
+                <div class="grey-space grey-border dashed-border-green"></div>
+                <div class="main-content grey-border dashed-border-green">
```

Comment:
nice use of reusable grey-border class

Reply:
You could maybe use the `main` tag for the main content part of the page instead of `div`

Reply:
I'll look to change some of the divs and sections to more specific tags, main, aside etc.

the dashed-border boxes were used to show the grid structure before the content was added. The classes were commented out when they were not necessary anymore